src/components/cta.tsx

```diff
@@ -11,8 +11,8 @@ export const CallToAction: FC = () => (
 				rel="noreferrer"
 			>
 				<span className="hidden sm:block">GitHub</span>
-				<span className="sm:hidden">
-					<ExternalLinkIcon size={16} />
+				<span className="sm:ml-2">
+					<ExternalLink size={16} />
```

Comment:
```suggestion
					<ArrowUpRight size={16} />
```
The icon does not align with the mockups.

Reply:
Its fine the external link looks better than the arrow icon.